src/main/java/liquibase/ext/mongodb/statement/CreateCollectionStatement.java

```diff
@@ -54,20 +59,20 @@ public String getCommandName() {
 
     @Override
     public String toJs() {
-        return
-                "db."
-                        + getCommandName()
-                        + "("
-                        + getCollectionName()
-                        + ", "
-                        + ofNullable(options).map(Document::toJson).orElse(null)
-                        + ");";
+        return String.format("db.runCommand(%s)", createCommand().toString());
     }
 
     @Override
     public void execute(final MongoConnection connection) {
-        final CreateCollectionOptions createCollectionOptions = BsonUtils.orEmptyCreateCollectionOptions(options);
-        connection.getDatabase().createCollection(getCollectionName(), createCollectionOptions);
+        Bson bson = createCommand();
+        connection.getDatabase().runCommand(bson);
     }
 
-}
+    private Bson createCommand() {
+        Document commandOptions = new Document(COMMAND_NAME, getCollectionName());
+        if(options!=null) {
+            commandOptions.putAll(options);
+        }
+        return commandOptions.toBsonDocument(Document.class, getDefaultCodecRegistry());
+    }
```

Comment:
`create:<collectionName>` needs to be the first entry in the Document, followed by any options if present.